### DISCUSSION TITLE
S905L的设备没有一个独立的类目呀

### DISCUSSION THREAD

**Discussion by dz515704201** [ophub/amlogic-s9xxx-armbian#1458] [Amlogic]
至今还没人填这个空白么，我相信有人用类似的固件也刷上去了吧

**Comment by ophub** [comment]
看起来这是一项国际空白

**Reply by dz515704201** [reply]
大佬亲自回复我受宠若惊，要是我有什么地方理解不到位的请谅解。不过确实也有能搜到成功刷上了你这个固件的帖子，只是没把整合这里

**Comment by shinyicn** [comment]
支持，我的一天上海贝尔联通版，刷机老是进不了U盘启动。
